create/how_to/updating.md

```diff
@@ -114,6 +114,10 @@ In this guide, we will:
 
 ### Step 1: Set all fields as displayed attributes
 
+::: note
+This step is not mandatory when creating dumps in MeiliSearch v0.21 and above.
+:::
+
```

Comment:
this whole step was added due to a bug that is now fixed in v0.21.0 :)

Reply:
I think you can completely delete this part

Reply:
You don't need to put your attributes as displayed anymore.

Reply:
As discussed, we will keep this note and the first step as they are for users migrating from v0.20 to v0.21.

We will probably replace both with a very short note in a future release.